pkg/aws/policies.go

```diff
@@ -690,6 +690,47 @@ func isOCMRole(roleName *string) bool {
 	return strings.Contains(aws.StringValue(roleName), fmt.Sprintf("%s-Role", OCMRole))
 }
 
+// isUserRole checks the role tags in addition to the role name, because the word 'user' is common
+func (c *awsClient) isUserRole(roleName *string) (bool, error) {
+	if strings.Contains(aws.StringValue(roleName), OCMUserRole) {
+		roleTags, err := c.iamClient.ListRoleTags(&iam.ListRoleTagsInput{
```

Comment:
Would you mind moving all the role related code into a different file than policies. 

Also do we need to list role Tags or are we getting role.tags now from aws?

Reply:
Sure, I'll create a sequence PR to refactor to a new file, as some role-related functions are already merged with **rosa list ocm-roles** PR.
Unfortunately, when calling `c.iamClient.ListRolesPages` we don't get the tags,
only when retrieving a single role with `c.iamClient.GetRole` we do get also the tags in the response.